### DISCUSSION TITLE
Dotts about license

### DISCUSSION THREAD

**Discussion by sidivalfn** [community/community#134842] [Enterprise]
### Select Topic Area

Question

### Body

Good morning, I need to open a repository for the company That I work for, but I'm having difficulties to understand the documentation, what I need to know is: To have a private repository is it necessary to have a paid license? In this repository I wanted only 1 person to have access or see the content.

**Comment by ghost** [comment]
The user needs a license to have an account to be able to see/read the repository when using Github enterprise.
